configure

```diff
@@ -576,11 +576,12 @@ def get_version_helper(cc, regexp):
 
 def get_llvm_version(cc):
   return get_version_helper(
-      cc, r"(^clang version|based on LLVM) ([3-9]\.[0-9]+)")
+    cc, r"(^clang version|^FreeBSD clang version|based on LLVM) " +
```

Comment:
I did not think of a non-capturing parentheses. It does not break existing check and looks better. Thanks.
@bnoordhuis I fixed in ed1d266f36e5f7717bac4a2ec19bf1878afd82ec. PTAL.